CSS/home.css

```diff
@@ -0,0 +1,1030 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+}
+
+body {
+  display: grid;
+  grid-template-columns: auto;
+  background-color: #fff;
+  overflow-x: hidden;
+}
+
+/* Hero Section Starts */
+
+#hero {
+  width: 100vw;
+}
+
+.header-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.pre-header {
+  display: none;
+}
+
+#main-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  height: 73px;
+  background: #fff;
+  box-shadow: 4px 4px 4px 0 rgba(0, 0, 0, 0.25);
+  position: relative;
+}
+
+.main-header-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  height: 73px;
+  background: #fff;
+  position: relative;
+}
+
+.hamburger-container {
+  width: 31px;
+  height: 27px;
+  margin-left: 23px;
+  transition: 0.1s;
+}
+
+#hamburger-link {
+  width: 100%;
+  height: 100%;
+}
+
+.hamburger {
+  width: 100%;
+  height: 100%;
+}
+
+.hamburger-container:hover {
+  transform: scale(1.05);
+}
+
+.logo-container {
+  max-width: 130px;
+  max-height: 51px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+.logo-link {
+  width: 100%;
+  height: 100%;
+}
+
+.logo {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
- [ ] I really appreciate your efforts for using a logo for the mobile version 🙌🙌 However, please note that the logo is not present in the design guidelines for mobile version. Please fix this for getting a design close to the given guidelines.

_Kindly apply this to both pages_